acceptance-tests/src/test/resources/features/UKDrivingLicence/DrivingLicenceCRIAPI.feature

```diff
@@ -6,7 +6,7 @@ Feature: DrivingLicence CRI API
     Given Driving Licence user has the user identity in the form of a signed JWT string for CRI Id driving-licence-cri-shared-dev and row number 6
     And Driving Licence user sends a POST request to session endpoint
     And Driving Licence user gets a session-id
-    When Driving Licence user sends a POST request to Driving Licence endpoint using jsonRequest DVLAValidJsonPayload
+    When Driving Licence user sends a POST request to Driving Licence endpoint using jsonRequest DVLAValidJsonPayload and document checking route is dcs
```

Comment:
We shouldnt need this. The absence of a route is what is happening now and in this case we should be hitting DCS by default